test/size.txt

```diff
@@ -1,39 +1,38 @@
-      max   |         min   |       gzip   |   file
-      ---   |         ---   |        ---   |   ---
- 49.49 kB   |      5.5 kB   |    2.19 kB   |   alp.js / alp.max.js
-651.01 kB   |   166.19 kB   |   46.47 kB   |   v0.js / amp.js
- 224.2 kB   |    38.54 kB   |   12.31 kB   |   v0/amp-access-0.1.js
- 27.75 kB   |     6.07 kB   |     2.4 kB   |   v0/amp-accordion-0.1.js
-240.38 kB   |    57.03 kB   |   19.85 kB   |   v0/amp-analytics-0.1.js
- 89.86 kB   |     9.53 kB   |    3.64 kB   |   v0/amp-anim-0.1.js
-  86.7 kB   |     7.24 kB   |    2.93 kB   |   v0/amp-audio-0.1.js
- 81.55 kB   |     8.36 kB   |    3.18 kB   |   v0/amp-brid-player-0.1.js
- 99.81 kB   |     8.07 kB   |    3.09 kB   |   v0/amp-brightcove-0.1.js
- 193.7 kB   |     33.2 kB   |    9.55 kB   |   v0/amp-carousel-0.1.js
- 74.13 kB   |     6.99 kB   |     2.8 kB   |   v0/amp-dailymotion-0.1.js
- 87.24 kB   |      5.3 kB   |    2.28 kB   |   v0/amp-dynamic-css-classes-0.1.js
-139.52 kB   |    15.32 kB   |    6.06 kB   |   v0/amp-facebook-0.1.js
- 34.94 kB   |     6.75 kB   |    2.66 kB   |   v0/amp-fit-text-0.1.js
- 96.48 kB   |     9.68 kB   |    3.51 kB   |   v0/amp-font-0.1.js
-134.03 kB   |    15.46 kB   |    5.61 kB   |   v0/amp-iframe-0.1.js
- 225.1 kB   |    40.05 kB   |   11.23 kB   |   v0/amp-image-lightbox-0.1.js
- 94.58 kB   |     7.77 kB   |    3.07 kB   |   v0/amp-instagram-0.1.js
- 83.57 kB   |     8.64 kB   |     3.5 kB   |   v0/amp-install-serviceworker-0.1.js
- 80.98 kB   |     7.82 kB   |    3.05 kB   |   v0/amp-jwplayer-0.1.js
-105.13 kB   |     8.78 kB   |     3.4 kB   |   v0/amp-kaltura-player-0.1.js
-139.39 kB   |    15.37 kB   |    4.84 kB   |   v0/amp-lightbox-0.1.js
-105.78 kB   |     9.07 kB   |    3.42 kB   |   v0/amp-list-0.1.js
-144.21 kB   |    40.51 kB   |   14.28 kB   |   v0/amp-mustache-0.1.js
-109.35 kB   |    20.49 kB   |    6.06 kB   |   v0/amp-pinterest-0.1.js
- 73.24 kB   |     6.56 kB   |    2.63 kB   |   v0/amp-reach-player-0.1.js
- 97.65 kB   |    11.32 kB   |    4.02 kB   |   v0/amp-sidebar-0.1.js
-108.47 kB   |    12.78 kB   |    4.56 kB   |   v0/amp-slides-0.1.js
-114.39 kB   |    13.24 kB   |    4.89 kB   |   v0/amp-social-share-0.1.js
- 73.82 kB   |     6.69 kB   |    2.67 kB   |   v0/amp-soundcloud-0.1.js
- 81.79 kB   |     8.54 kB   |    3.16 kB   |   v0/amp-springboard-player-0.1.js
-139.99 kB   |    15.44 kB   |     6.1 kB   |   v0/amp-twitter-0.1.js
-106.05 kB   |    11.33 kB   |    4.13 kB   |   v0/amp-user-notification-0.1.js
- 73.75 kB   |     6.66 kB   |    2.67 kB   |   v0/amp-vimeo-0.1.js
- 73.28 kB   |     6.52 kB   |    2.63 kB   |   v0/amp-vine-0.1.js
- 84.28 kB   |     8.87 kB   |     3.4 kB   |   v0/amp-youtube-0.1.js
- 147.9 kB   |    33.68 kB   |   11.64 kB   |   current-min/f.js / current/integration.js
\ No newline at end of file
+      max   |        min   |                                      gzip   |   file
```

Comment:
Please drop this file. We usually have a cron job that will update this file for us. Also the numbers seem wrong and bloated - my guess these are not the closure-compiled numbers.

cc/ @erwinmombay